components/blocks/Run/Run.vue

```diff
@@ -27,19 +20,13 @@
       </div>
       <p>{{ user }}</p>
     </div>
-    <div
-      class="
-        flex flex-col
-        text-right
-        md:text-center md:flex-row
-        gap-x-5
-        flex-grow
-      "
-    >
-      <p class="md:flex-grow">{{ time }}</p>
+    <div class="run__info">
+      <p>{{ time }}</p>
+      <!--      TODO: Need good BEM name for this -->
       <div class="hidden md:flex flex-row md:flex-grow">
         <p>{{ platform }}</p>
       </div>
+      <!--      TODO: Need good BEM name for this -->
```

Comment:
Don't forget about these comments

Reply:
oops

Reply:
this one is annoying because i don't know how to refer to this one at all. its a div that lets the run row show different things on different devices

Reply:
Yeah, I'm not sure either. I think it's probably fine as is, since it's nested within the greater `run__info` class.

Reply:
I'd say it's okay too. Resolving